_source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Ship Kubernetes metrics over Helm with the OpenTelemetry Collector
+logo:
+  logofile: k8s-helm.svg
+  orientation: horizontal
+data-source: Kubernetes over Helm
+open-source:
+  - title: 
+    github-repo: 
+templates: ["docker"]
```

Comment:
@Doron-Bargo & @yotamloe 

For lines 6-9 and 15: What should the values be for these metadata fields?

Reply:
This should be the url after we merge: [https://github.com/logzio/logzio-helm/tree/master/charts/opentelmetry](https://github.com/logzio/logzio-helm/tree/master/charts/opentelmetry)